packages/fast-components-react-base/src/auto-suggest/examples.data.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
-import schema from "./auto-suggest.schema.json";
-import listboxItemSchema from "../listbox-item/listbox-item.schema.json";
+import { autoSuggestSchema, listboxItemSchema } from "../";
```

Comment:
This path may have issues resolving in certain environments. Can we update the pathing.

Reply:
to "../index.ts" ?

Reply:
yeah, I would path it directly to be safe